client/catapult/plugins/txes/restriction_mosaic/tests/state/RestrictionValueMapTests.cpp

```diff
@@ -142,14 +142,17 @@ namespace catapult { namespace state {
 	TEST(TEST_CLASS, CanRemoveValueNotInMap) {
 		// Arrange:
 		RestrictionValueMap<uint64_t> map;
+		map.set(111, 444);
+		map.set(321, 987);
 
 		// Act:
-		map.remove(111);
+		map.remove(222);
```

Comment:
workaround array-bounds. it does not follow if in remove and warns calling erase on empty vector.